src/lib/index.ts

```diff
@@ -31,33 +50,39 @@ class TPAStylePlugin {
 
   getCompilationHash() {
     if (isWebpack5) {
-      return createHash("md5").update(this._options.packageName).digest("hex");
+      return createHash('md5')
+        .update(this._options.packageName)
+        .digest('hex');
     }
 
-    return createHash("md5")
+    return createHash('md5')
       .update(new Date().getTime().toString())
-      .digest("hex");
+      .digest('hex');
   }
 
-  apply(compiler) {
+  apply(compiler: webpack.Compiler) {
     const cheapModuleEvalSourceMap = isWebpack5 ? 'eval-cheap-module-source-map' : 'cheap-module-eval-source-map';
     const shouldEscapeContent = [cheapModuleEvalSourceMap, 'cheap-eval-source-map'].includes(
-      compiler.options.devtool
+      compiler.options.devtool as string
```

Comment:
`devtool` config could be boolean as well. But at Wix we do not use such. So casting to string explicitly.